src/sentry/models/latestappconnectbuildscheck.py

```diff
@@ -0,0 +1,50 @@
+"""Database models to keep track of the App Store Connect builds for a project.
+
+For projects using the App Store Connect symbol source this keeps track of the last time we
+did manage to check for available builds.
+"""
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model
+
+
+class LatestAppConnectBuildsCheck(Model):
+    """
+    The last date and time Sentry checked App Store Connect for new builds associated with a
+    specific appconnect source in a project.
+    """
+
+    __include_in_export__ = False
+
+    project = FlexibleForeignKey("sentry.Project", db_constraint=False)
+
+    # The symbol source's unique identifier in the project. Also known as the id on an
+    # AppStoreConnectConfig.
+    source_id = models.CharField(max_length=200)
+
+    # When sentry last checked App Store Connect for new builds. A check may return zero new builds,
+    # or no builds at all if all existing builds have expired.
+    last_checked = models.DateTimeField(default=timezone.now)
+
```

Comment:
I'd recommend you add a `date_added` field. It just tends to be useful for debugging. If you subclass `DefaultFieldsModel` it'll add `date_added` and `date_updated` for you automatically, or if you prefer to not have `date_updated` you can just add the field separately.

Reply:
(You'll need to regenerate your migration after adding the default model)

Reply:
oh dang it i was hoping i could skip that, thanks!